EXTRA_FIM/datautils/plotting.py

```diff
@@ -159,7 +159,7 @@ def waves_figure(
     # --- compute and plot extrapolated wave
     if compute_extra:
         print("Computing EXTRA wave...")
-        from ..EXTRA_FIM.extra import extra_waves
+        from EXTRA_FIM.extra import extra_waves
```

Comment:
extra.py resides in the superdirectory, so this should be
```suggestion
        from ..extra import extra_waves
```

Using relative locations for imports inside the module has the huge advantage that it will work for both EXTRA_FIM.EXTRA_FIM and EXTRA_FIM imports, depending on how the search paths are set by the user.